modules/reference/pages/internal-metrics-reference.adoc

```diff
@@ -145,6 +179,26 @@ Number of compacted segments.
 
 ---
 
+=== vectorized_storage_log_compaction_bytes_removed
+
+Number of bytes removed from local storage by compaction operations. This is tracked on a per-topic and per-partition basis.
+
+This metric resets every time partition leadership changes. It primarily exists to track the fact that compaction is performing operations on local storage.
+
```

Comment:
```suggestion
It primarily exists to track the fact that compaction is performing operations on local storage.
```

My bad, the description I originally gave was inaccurate.
The metric resets when the replica is moved or other maintenance ops (automatic or manual), but this fact is true for many metrics. so it's probably better to leave this part out. During normal operations, the metric tracks the compaction of the replica of this partition

Reply:
Fixed!